e2e/tests/functional/plugins/lad/lad.e2e.spec.js

```diff
@@ -0,0 +1,131 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+const { test, expect } = require('../../../../pluginFixtures');
+const { createDomainObjectWithDefaults } = require('../../../../appActions');
+
+test.describe('Testing LAD table @unstable', () => {
+    test('telemetry value exactly matches latest telemetry value received in real time', async ({ page, openmctConfig }) => {
+        await page.goto('./', { waitUntil: 'networkidle' });
```

Comment:
navigate directly to realtime instead of home + time conductor click

Reply:
I don't know if this is possible, but I'm using an imported function to clean it up a bit (though it does the same thing)

Reply:
It works if you provide the tc.mode, and offsets / bounds values but won't switch mode unless it has all three query parameters. It also doesn't default to current bounds/offsets if they aren't provided in the query parameters (meaning that we would have to manually calculate bounds).

We should fix up the router logic so that we only have to provide the mode, and then update the helper functions to switch modes using hash url navigation only